src/app/mapper.service.ts

```diff
@@ -0,0 +1,91 @@
+import { Injectable } from '@angular/core';
+import { Repository } from './repository';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class MapperService {
+  constructor() { }
+
+  mapRepos(repos: any[]): Repository[] {
```

Comment:
Non-blocking and more for future reference - generally, using `any` as a type is discouraged since it bypasses type checking (which sort of defeats the whole purpose of using Typescript). It looks like you may be able to install this library to add types to fields in API responses: https://www.npmjs.com/package/@octokit/types

Reply:
Ah you're right, that was just for REST API endpoints 😬 but yeah anyways, just something to keep in mind if you stumble across having to use `any` as a type!